### PR TITLE
Add ukrainian translation

### PR DESCRIPTION
**Pull request by yanicka**

### CODE REVIEW

File: lang/uk.po [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+msgid "ERR_LICENCES_FAILED\n"
+msgstr "vrms-rpm: не вдалося прочитати перелiк припустимих ліцензій\n"
+
+msgid "ERR_LICENCES_BADFILE\n"
+msgstr "vrms-rpm: не вдалося прочитати перелiк припустимих ліцензій iз \"%s\"\n"
+
+msgid "ERR_BADOPT_COLOUR\n"
+msgstr  "vrms-rpm: аргумент для флага --colour може бути\n"
+        "одним з наступних значень: 'no', 'yes', або 'auto'\n"
```
Comment (suve) [inline lang/uk.po]: The error message strings should not contain newlines (`\n`).

File: lang/uk.po [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+msgid "ERR_BADOPT_LIST\n"
+msgstr  "vrms-rpm: аргумент для флага --list може бути одним\n"
+        "з наступних значень: 'none', 'non-free', 'free' або 'all'\n"
```
Comment (suve) [inline lang/uk.po]: Please don't put newlines (`\n`) in the error message strings.

File: lang/uk.po [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n==1 ? 0 : 1);\n"
+
+msgid "TRANSLATION_AUTHOR\n"
+msgstr "На українську мову переклала yanicka\n"
+
+msgid "FREE_PACKAGES_COUNT\n"
+msgid_plural "FREE_PACKAGES_COUNT\n"
+msgstr[0] "%d вільного пакету (%s)\n"
+msgstr[1] "%d вільних пакетiв (%s)\n"
```
Comment (suve) [inline lang/uk.po]: Just to make sure - does Ukrainian use only two forms? If you have more plural forms, you can edit line 6 (`Plural-Forms: ...`) to reflect this.

The manual for GNU gettext says that Ukrainian typically uses three plural forms.
https://www.gnu.org/software/gettext/manual/html_node/Plural-forms.html

File: man/uk.man [hunk 4]
```diff
@@ -0,0 +1,91 @@
+.TH VRMS-RPM 1 "2019-10-07"
+.SH НАЗВА
+vrms-rpm - звіт щодо встановленого пропрієтарного ПЗ
+
+.SH СИНОПСИС
+\fBvrms-rpm\fR \fI[OPTION]\fR ...
```
Comment (suve) [inline man/uk.man]: Please translate `OPTION` here.

File: lang/uk.po (hunk 1 above)
Comment (yanicka) [reply lang/uk.po]: Do you mean I should delete the newline (\n) in the first line(e.g. 92), at the end of the string(e.g. 93) or both?

Comment (suve) [reply lang/uk.po]: The message should have only a single newline at the very end, so option 1).

File: lang/uk.po (hunk 3 above)
Comment (yanicka) [reply lang/uk.po]: I just saw the same "mistake" in the Russian translation ru.po. I edited it too.

**Comment by suve** [comment]
There were some Unicode quotes in the man page, but that's a simple enough issue so I replaced them manually.

Apart from that, everything looks good. Merged. Thanks!

**Pull request merged**